packages/bitcore-client/src/wallet.ts

```diff
@@ -442,6 +450,14 @@ export class Wallet {
     await this.saveWallet();
   }
 
+  async rmToken({ tokenName }) {
+    if (!this.tokens) {
+      return;
+    }
+    this.tokens = this.tokens.filter(tok => tok.name !== tokenName || (!tok.name && tok.symbol !== tokenName));
```

Comment:
If `tok.name` is undefined, this won't reach the or. This works for me:

`this.tokens = this.tokens.filter(tok => (tok.name && tok.name !== tokenName) || ( tok.symbol !== tokenName));`